src/app/utils/ValidatorUtils.ts

```diff
@@ -289,6 +289,17 @@ export class ValidatorUtils extends CacheModule {
 		return result
 	}
 
+	async areRocketpoolValidatorsSubscribed(): Promise<boolean> {
+		const validators = await this.getAllValidatorsLocal()
```

Comment:
Use `getAllMyValidators()` instead. Local validators may not be populated with rocketpool statistics. 
getAllMyValidators() populates this data into the object

Reply:
This would cause another API request, see [HERE](https://github.com/gobitfly/eth2-beaconchain-explorer-app/blob/7cfaec4df03fd3477737c3bac86abef33df2e0db/src/app/utils/ValidatorUtils.ts#L372). Are we fine with that?

Alternatively:
What about using `getDashboardDataValidators`? We could save a flag "areRocketpoolValidatorsSubscribed" in `ValidatorUtils` and use that?

Reply:
Oh boy, my bad, changed.